server/src/main/kotlin/suwayomi/tachidesk/manga/MangaAPI.kt

```diff
@@ -107,6 +107,7 @@ object MangaAPI {
         }
 
         path("download") {
+            get("bulk", DownloadController::bulkQueueChapter)
```

Comment:
Why not something like `{mangaId}/download/bulk`? and use a List<Int>

Reply:
You can select multiple manga and queue for download in Tachiyomi, this takes care of that as well.

Reply:
I would suggest something like
```kotlin
data class MangaBulkDownload(
    val mangaId: Int
    val all: Boolean?,
    val startIndex: Int,
    val endIndex: Int
)
```
Since in places where we would want bulk download we wouldn't have the chapter list available.